modules/identity/src/Volo.Abp.Identity.EntityFrameworkCore/Volo/Abp/Identity/EntityFrameworkCore/EfCoreIdentityUserRepository.cs

```diff
@@ -174,13 +174,14 @@ into gp
         DateTime? minModifitionTime = null,
         CancellationToken cancellationToken = default)
     {
+        var upperFilter = filter?.ToUpper();
```

Comment:
I think we can use the `ToUpperInvariant()` to prevent character mismatches and pass [The Turkey Test](http://www.moserware.com/2008/02/does-your-code-pass-turkey-test.html).